libs/aws/tests/test_aws.py

```diff
@@ -111,3 +111,47 @@ def test_get_queue_attributes(aws_env):
     for att in valid_attributes:
         att_returned = get_queue_attributes(queue_name="queue1", attribute=att)
         assert att_returned.get(att)
+
+
+@mock_sqs
+def test_get_queue_attributes(aws_env):
+    resource = boto3.resource("sqs")
+
+    resource.create_queue(QueueName="queue1")
+
+    valid_attributes = [
+        'ApproximateNumberOfMessages',
+        'ApproximateNumberOfMessagesDelayed',
+        'ApproximateNumberOfMessagesNotVisible',
+        'CreatedTimestamp',
+        'DelaySeconds',
+        'LastModifiedTimestamp',
+        'MaximumMessageSize',
+        'MessageRetentionPeriod',
+        'QueueArn',
+        'ReceiveMessageWaitTimeSeconds',
+        'VisibilityTimeout'
+    ]
+
+    all_attributes = get_queue_attributes(queue_name="queue1")
+
+    assert len(all_attributes) == len(valid_attributes)
+
+    for att in valid_attributes:
+        att_returned = get_queue_attributes(queue_name="queue1", attribute=att)
+        assert att_returned.get(att) is not None
+
+
+@mock_sqs
+def test_get_missing_queue_attribute(aws_env):
+    """
+    When the attribute isn't found in the queue boto 3 returns the attribute with value None
+    """
+    resource = boto3.resource("sqs")
+
+    resource.create_queue(QueueName="queue1")
+
+    # FifoThroughputLimit is an attribute specific for FIFO queues
+    att = 'FifoThroughputLimit'
```

Comment:
Err, why not just use "FakeAttribute"
```suggestion
    att = 'FakeAttribute'
```

Reply:
Because I wanted to show that you might use a valid attribute but because this isn't a FIFO queue, it won't have that and will generate an exception